addons/account/models/account_account.py

```diff
@@ -859,13 +862,12 @@ def _merge_method(self, destination, source):
 class AccountGroup(models.Model):
     _name = "account.group"
     _description = 'Account Group'
-    _parent_store = True
     _order = 'code_prefix_start'
     _check_company_auto = True
     _check_company_domain = models.check_company_domain_parent_of
 
     parent_id = fields.Many2one('account.group', index=True, ondelete='cascade', readonly=True, check_company=True)
-    parent_path = fields.Char(index=True)
+    parent_path = fields.Char(index=True)  # unused, can be removed
```

Comment:
Since we are in master, we can do it

Reply:
I prefer to do that in another pr for a a faster merge and easier backport. (justified in commit message)